src/main/java/tutorial/Game.java

```diff
@@ -46,13 +53,18 @@ private boolean isComplete() {
         return towerOne.getDiscs().isEmpty() && towerTwo.getDiscs().isEmpty();
     }
 
+    //parameter types?
+    //negated method meaning?
     private boolean moveIsIllegal(int fromTower, int toTower) {
+        //could this not be a method on the tower?
         return getTower(fromTower).getDiscs().isEmpty() ||
                 (!getTower(toTower).getDiscs().isEmpty() &&
+                        //need to expose disc?
                         getTower(fromTower).getDiscs().peek().getSize() >
                                 getTower(toTower).getDiscs().peek().getSize());
     }
 
+    //legibility?
     private void adjustMovesIfOdd(int numberOfDiscs) {
```

Comment:
Whilst I appreciate this saves some repetition - I think it would be more legible if we just had an explicit list of moves for odd and moves for even.